_includes/docs/latest/setup-cloudsql.md

```diff
@@ -26,10 +28,27 @@ To set up your Cloud SQL instance for Forseti, follow the steps below:
     to proxy your connection to your Cloud SQL instance. Your
     INSTANCE_CONNECTION_NAME is the **Instance Connection Name** under
     **Properties** on the Cloud SQL dashboard instance details, in the format "PROJECTID:REGION:INSTANCEID".
+    
+    If you are using Cloud SDK authentication:
 
       ```bash
       <path/to/cloud_sql_proxy> -instances=INSTANCE_CONNECTION_NAME=tcp:3306
       ```
+    
+    If you are using a service account to authenticate (recommended for production environments):
+    
+      ```bash
+      <path/to/cloud_sql_proxy> \
+                  -instances=<INSTANCE_CONNECTION_NAME>=tcp:3306 \
```

Comment:
nit: This indent looks really wide, please keep the indent 4 spaces from the previous line

```bash
<path/to/cloud_sql_proxy> \
    -instances=<INSTANCE_CONNECTION_NAME>=tcp:3306 \
    -credential_file=<PATH_TO_KEY_FILE>
```